va/va_dec_hevc.h

```diff
@@ -481,6 +481,10 @@ typedef struct  _VASliceParameterBufferHEVC
     int8_t                  ChromaOffsetL1[15][2];
     /** HEVC syntax element. */
     uint8_t                 five_minus_max_num_merge_cand;
+    /** HEVC syntax element. */
+    uint8_t                 num_entry_point_offsets;
+    /** HEVC syntax element. */
+    uint8_t                 entry_offset_to_subset_array;
```

Comment:
The types of these elements don't look right (uint8_t -> max value 255).

Reply:
@fhvwy yeah, i checked that the type of these elements are uint16_t